### ISSUE TITLE
Clearer documentation for prep_riboviz.py call

### ISSUE THREAD

**Issue by ewallace** [riboviz/RiboViz#80]
I'm finding the updated prep_riboviz.py call confusing because (a) it's different from what it used to be (b) it could be more clearly and prominently explained than currently in `docs/run-vignette.md`.

Can we put an anatomy of the **command** 
```
python -m riboviz.tools.prep_riboviz rscripts/ vignette/vignette_config.yaml
```
and options such as `--dry-run`, somewhere very easy to find. Either at the top of `run_vignette.md` in a section "How to call prep_riboviz", or in its own help page? 

(What's the python equivalent of a help file?)

Apologies if this documentation already exists and I didn't find it.

**Comment by kavousan** [comment]
A code review by @ewallace and @FlicAnderson will be good.

**Comment by mikej888** [comment]
Branch [docs-80](https://github.com/riboviz/RiboViz/tree/docs-80) branched from [use-umi-tools-deplex-81](https://github.com/riboviz/RiboViz/tree/use-umi-tools-deplex-84) has rewritten documentation. See [README.md](https://github.com/riboviz/RiboViz/blob/docs-80/README.md) and links to:

* Map mRNA and ribosome protected reads to transcriptome and collect data into an HDF5 file. This describes how to run vignette without detailed exp;anations of configuration or what is going on.
* Run UMI extraction, deduplication and demultiplexing examples. This page now solely describes how to run these examples.
* What the RiboViz workflow does. This describes in detail what `prep_riboviz.py` does, including deduplication and demultiplexing, and the files it outputs.
* Configuring the RiboViz workflow. This describes the required inputs and the parameters of the YAML configuration file.
* Running the RiboViz workflow. This describes in detail how to run `prep_riboviz.py`, without any reference to the vignette.

**Comment by ewallace** [comment]
Request new riboviz users @amandamok and @john-s-f check documentation and suggest changes where needed.

**Comment by ewallace** [comment]
> If relative then they are relative to RiboViz home directory.

Really? Not relative to the directory `prep_riboviz.py` is called from?

**Comment by mikej888** [comment]
Closed by  #118
